src/Asset/Widget/Queue.php

```diff
@@ -156,23 +156,23 @@ public function countItemsInQueue($location, $type = 'frontend')
      *
      * @return string|null
      */
-    public function render($location, $type = 'frontend')
+    public function render($location, $type = 'frontend', $wrappertemplate = '', $holdertemplate = '')
     {
```

Comment:
Small favour… can we go for camelCase on `$wrapperTemplate` and `$holderTemplate` to keep in-line with PSR-2 and the way we're taking our code standards.